specreduce/background.py

```diff
@@ -341,6 +341,12 @@ def bkg_spectrum(self, image=None):
             (spatial) direction is axis 0 and dispersion (wavelength)
             direction is axis 1. If None, will extract the background
             from ``image`` used to initialize the class. [default: None]
+        bkg_statistic : str, optional
+            Statistical method used to collapse the background image.
+            Supported values are:
+            - `'median'` : Uses the median (`np.nanmedian`).
```

Comment:
Ref linking needs full name of the package. Please fix accordingly to get rid of warnings on ReadTheDocs. Thanks!
```suggestion
            - `'median'` : Uses the median (`numpy.nanmedian`).
```

Reply:
I also wonder if passing in the function directly is more straightforward than using string for translation. It certainly would be more flexible and not confined to numpy. User can pass in their custom stats func... but is that too dangerous?

Reply:
I was thinking about the implementation in Jdaviz (passing the traitlet value for Statistic) but can see the applicability to be able to pass the function directly too. Could modify the jdaviz logic to make this possible so I'm open to either way.

Reply:
I don't think the API here should be dictated by what Jdaviz does. The API we pick should be useful to someone who is using just specreduce natively. Jdaviz can hook into it downstream no matter what.

Reply:
I've made the other changes locally, I'll leave the string in for now but am more than open to modifying the logic here to make it more flexible

Reply:
Let's see what specreduce maintainers say. Thanks!

Reply:
i like the idea of passing in a function for full generality, but also recognize that in this context, mean and median are the most applicable options in the the vast majority of cases.